services/mock-service/src/fsp-integration/intersolve-visa/intersolve-visa.mock.service.ts

```diff
@@ -578,7 +582,29 @@ export class IntersolveVisaMockService {
     };
   }
 
-  public transfer(fromToken: string): IntersolveVisaMockResponseDto {
+  public transfer(
+    fromToken: string,
+    amount: number,
+  ): IntersolveVisaMockResponseDto {
+    if (amount === MAGIC_FAIL_OPERATION_REFERENCE_AMOUNT) {
+      return {
+        status: HttpStatus.NOT_FOUND,
+        statusText: 'Not Found',
```

Comment:
When I tested this, I received a 400 Bad request. Is there a specific reason we're not mirroring this behaviour?
If that's the case you can resolve this comment and merge it in.

Reply:
No reason I'll change it :)